src/main/flight/pid.c

```diff
@@ -420,6 +420,17 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     // Dynamic ki component to gradually scale back integration when above windup point
     const float dynKi = MIN((1.0f - motorMixRange) * ITermWindupPointInv, 1.0f);
 
+    // rotate old I to the new coordinate system
+    const float gyroToAngle = dT * RAD;
```

Comment:
Not really necessary as compiler will figure it out and compute only once. Leave it that way for now though.